spacemacs/packages.el

```diff
@@ -1677,14 +1677,19 @@ If ARG is non nil then `ag' and `pt' and ignored."
           (call-interactively (spacemacs//helm-projectile-do-search-find-tool
                                tools))))
 
+      (setq helm-projectile-sources-list '(helm-source-projectile-projects
+                                           helm-source-projectile-files-list
+                                           helm-source-recentf
+                                           helm-source-pp-bookmarks
+                                           helm-source-file-cache))
```

Comment:
You may want to have a look at [CtrlP relevant features](http://kien.github.io/ctrlp.vim/):

- Built-in Most Recently Used (MRU) files monitoring and search.

- Built-in project's root finder *(which means you can search for files in current project)*.

- Optional cross-sessions caching and history allow for fast initialization *(essentially the file cache feature).